src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.WIN32_FIND_DATA.cs

```diff
@@ -22,6 +22,9 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint dwReserved1;
             private fixed char _cFileName[MAX_PATH];
             private fixed char _cAlternateFileName[14];
+            internal uint dwFileType;
```

Comment:
This is the bug. We seem to getting lucky and it does not seem to be causing any functional issues currently.

Reply:
False alarm. There is no actual bug. These fields are under `#ifdef _MAC` in the header files